### PR TITLE
Note addition

### PR DESCRIPTION
**Pull request by joinimran**
As suggested by the user, I have added a note for the user to route to a relevant information page.

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/8264

### CODE REVIEW

**Comment by joinimran** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/microsoft-defender-atp/mac-install-with-jamf.md [hunk 1]
```diff
@@ -30,6 +30,9 @@ ms.topic: conceptual
 
 Learn how to deploy Microsoft Defender ATP for macOS with Jamf Pro.
 
+> [!NOTE]
+> If you are using macOS Catalina (10.15.4) and newer versions of macOS see [New configuration profiles for macOS Catalina and newer versions of macOS](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/mac-install-with-jamf.md:34]:
```suggestion
> If you are using macOS Catalina (10.15.4) or newer versions of macOS, see [New configuration profiles for macOS Catalina and newer versions of macOS](https://docs.microsoft.com/windows/security/threat-protection/microsoft-defender-atp/mac-sysext-policies).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by joinimran** [comment]
Thanks @JohanFreelancer9 - @e0i - Ready to go live.

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by illfated — APPROVED** [review]
Looks good to me.

**Pull request merged**